src/arfsdao.ts

```diff
@@ -40,6 +38,9 @@ import {
 	ArFSPublicFolderTransactionData
 } from './arfs_trx_data_types';
 import { buildQuery } from './query';
+import { FsFile } from './fsFile';
+
+export const ArFS_O_11 = '0.11';
```

Comment:
This should no longer live in this file, looks like its leftover from a merge conflict